packages/superdough/superdough.mjs

```diff
@@ -260,7 +260,7 @@ export function resetGlobalEffects() {
   analysersData = {};
 }
 
-export const superdough = async (value, t, hapDuration) => {
+export const superdough = async (value, t, hapDuration, cps, cycle) => {
```

Comment:
could there be a default value for cycle to make am (and potential future friends) be relative to seconds? so am(4) would be 4Hz. thinking about using superdough without strudel, where cycle is a non existing concept

Reply:
yeah you could just pass in the time in seconds since clock start and it should work that way. I agree having both options would be nice for different effects